auxiliary_files/config_files/mumble-server.service.in

```diff
@@ -1,10 +1,10 @@
 [Unit]
-Description=Mumble Daemon
+Description=Mumble server
 After=network.target
 
 [Service]
 Type=exec
-ExecStart=/usr/bin/murmurd -fg -ini /etc/murmur.ini
+ExecStart=/usr/bin/@MUMBLE_SERVER_BINARY_NAME@ -fg -ini /etc/mumble-server.ini
```

Comment:
The use of `/usr` should be replaced with what users can set `CMAKE_INSTALL_PREFIX` to.